src/PrintCore/PrintCore.cs

```diff
@@ -753,6 +753,26 @@ public PMStatusCode SetDefault ()
 			}
 		}
 
+		[DllImport (Constants.PrintCoreLibrary)]
+		extern static IntPtr PMPrinterGetID (IntPtr printer);
+
+		public string ID {
+			get {
+				return CFString.FetchString (PMPrinterGetID (handle));
```

Comment:
Not , true as https://developer.apple.com/documentation/applicationservices/1459606-pmprintergetid?language=objc has:

```You should not release the string without first retaining it.```

Reply:
if you need to comment your own PR, before reviews, then it's a good sign those should be comment in the code ;-)